src/components/Permalink/index.tsx

```diff
@@ -89,6 +89,18 @@ export const Permalink: React.FC<PermalinkProps> = () => {
       );
     };
 
+    const updateLayerConfig = () => {
+      const externalLayers = map.getAllLayers().filter(l => l.get('isExternalLayer'));
```

Comment:
Why do we restore the opacity for external layers only?

Reply:
good point!

right now, because we only store the entire layer config for external layers. This requires a small refactoring though to harmonize `layers` and `customLayerParams`.

Reply:
I'll do this in a separate PR, because I have more ideas for a refactoring. generally my idea would be:

- use only one URL parameter to store layer information for both external and persisted layers
   - this includes visibility, opacity, isExternalLayer, groupName etc. 
- move `restoreTransientLayers` from here to PermalinkUtil
- if possible, move PermalinkUtil to https://github.com/terrestris/shogun-util since the permalink util functions are more closely related to SHOGun application state than to openlayers (in my opinion)